src/paho/mqtt/client.py

```diff
@@ -1734,7 +1764,10 @@ def _packet_read(self):
                     print(err)
                     return 1
                 else:
-                    byte, = struct.unpack("!B", byte)
+                    if len(byte) == 0:
+                        return 1
+                    byte = struct.unpack("!B", byte)
```

Comment:
Should I keep the following from `master` (https://github.com/eclipse/paho.mqtt.python/commit/2539ea2d3ab1bf10e5e255f66dc61c09b996633b)?

```
                    if len(byte) == 0:
                        return 1
```